### PR TITLE
CGSize extension added.

### PR DESCRIPTION
**Pull request by sereisoglu**
For use with withSize.

### CODE REVIEW

**Comment by sereisoglu** [comment]
keyboardWillShowNotification and keyboardWillHideNotification in LBTAFormController were removed in deinit.

**Comment by sereisoglu** [comment]
New init function added to LBTAListHeaderFooterController.

File: Source/Extensions/UIView+Stacking.swift [hunk 1]
```diff
@@ -62,6 +62,12 @@ extension UIView {
     }
 }
 
+extension CGSize {
+    static public func equalEdge(_ size: CGFloat) -> CGSize {
```
Comment (bhlvoong) [inline Source/Extensions/UIView+Stacking.swift]: You should rename this to equalEdge(_ edge: CGFloat).  "size" means a two dimensional W x H object in Swift whereas here it represents a single dimension.

**Comment by sereisoglu** [comment]
equalEdge (_ size: CGFloat) parameter name changed to equalEdge (_ edge: CGFloat)

File: Source/Extensions/UIView+Stacking.swift (hunk 1 above)
Comment (sereisoglu) [reply Source/Extensions/UIView+Stacking.swift]: I change it.

**Pull request merged**